tests/e2e/kubetest2-kops/deployer/up.go

```diff
@@ -146,39 +147,60 @@ func (d *deployer) createCluster(zones []string, adminAccess string, yes bool) e
 		}
 	}
 	if !foundCPCount {
+		// change this deprecated flag after the tester starts using kops.k8s.io/v1alpha3 APIs
 		args = appendIfUnset(args, "--master-count", fmt.Sprintf("%d", d.ControlPlaneCount))
 	}
 
+	// change this deprecated flag after the tester starts using kops.k8s.io/v1alpha3 APIs
 	args = appendIfUnset(args, "--master-volume-size", "48")
 	args = appendIfUnset(args, "--node-count", "4")
 	args = appendIfUnset(args, "--node-volume-size", "48")
-	args = appendIfUnset(args, "--set", adminAccess)
```

Comment:
What is the effect here?

Reply:
We already set that flag immediately after reading CreateArgs. The duplicate append was causing issues when I amended the `appendIfUnset` function